bfillings/vsearch.py

```diff
@@ -222,6 +222,43 @@ def getHelp(self):
         return help_str
 
 
+def parse_uc_to_clusters(uc_filepath, seed_id_field=8):
```

Comment:
true, there is clusters_from_uc_file() function in uclust.py and this works, I've removed parse_uc_to_clusters() and updated the tests to test parse_uc_to_clusters() with VSEARCH output